features/implicit_waiting.feature

```diff
@@ -9,7 +9,7 @@ Feature: Waiting for Elements Implicitly
     Then the slow element is waited for
     And implicit waits should be enabled
 
-  Scenario: Elements Collections Are Automatically Waited For
```

Comment:
I think this was called "Elements" not due to grammatical errors, but because it refers to the 'elements' method.

Reply:
Rightio. I'll reword this better